core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -1462,7 +1466,12 @@ fn handle_graphics_triangle_path<'gc>(
                 draw_triangle_internal(activation, triangle, drawing, culling);
             }
         } else {
-            panic!("Indices is not a vector");
+            avm2_stub_method!(
+                activation,
+                "flash.display.Graphics",
+                "drawGraphicsData",
+                "with null indices"
+            );
```

Comment:
Is this accepted in FP? If so, what happens then?

Reply:
Also, what if `vertices` is also null?

Reply:
If `indices` is null, then `vertices` is interpreted as a list of x,y coordinate pairs. I don't think `vertices` can be null - I'll improve te error message in a follow-up PR